app/views/admin/delete/owners-confirm.njk

```diff
@@ -26,7 +26,7 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
 
-    <form class="form" method="post">
+    <form class="form" method="post" onsubmit="confirmSubmit.disabled = true; return true;">
```

Comment:
Does this really work? THe 'confirmSubmit' name must match the name in the govukButton (line 42) - currently that is 'confirm' which is different